pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -249,8 +249,14 @@ def work(self):
 
         pokemon = pokemon_list[0]
 
+
+        min_ball = 1
+        #Load the min amount of ball from configuration file
+        if self.config['min_ball']:
```

Comment:
Thanks for your suggestion. I already pushed a new commit. I don't know how to merge 3 differents commits into a single one ;(